### ISSUE TITLE
Marketplace: Add education footers

### ISSUE THREAD

**Issue by allilevine** [Automattic/wp-calypso#61491]
### Details

We need to add some educational content to the /plugins homepage, in two footers.
<img width="773" alt="Screen Shot 2022-02-25 at 10 41 03 AM" src="https://user-images.githubusercontent.com/1689238/155743842-6a52b19a-4e7f-4bb7-ae89-f0243609f389.png">

For examples of components to re-use, see:
https://github.com/Automattic/wp-calypso/blob/c7e06d5e2fe1ca94643d16855209704c39452f30/client/my-sites/woocommerce/landing-page.tsx#L132

## Acceptance criteria

- On the plugins list screen, I can see a section at the bottom of the plugins home page that links off to relevant support articles (TBD)
- On the plugins list screen, I can see a section at the bottom of the home page that enforces why Dotcom is the right place to purchase plugins (Content TBD)

**Comment by allilevine** [comment]
@SaxonF Please drop links to relevant resources here 🙌 Also, is there an associated Figma?

**Comment by cpapazoglou** [comment]
Here are the figma designs f84fqtCG4pCnCh2Hq0But5-fi-590%3A11994.

Here is a thread regarding resources pdh6GB-gN-p2#comment-785

TL;DR; 
There aren't any articles ready yet. I think though that we can prepare these sections and launch the "Learn more" once we have at least one article.

**Comment by cpapazoglou** [comment]
Actually here are 3 articles that we might use pdh6GB-gN-p2#comment-812

**Comment by SaxonF** [comment]
Thanks for covering @cpapazoglou and sorry I missed this @allilevine ! I'm thinking, could we just start with the top learn more footer?

**Comment by allilevine** [comment]
> Thanks for covering @cpapazoglou and sorry I missed this @allilevine ! I'm thinking, could we just start with the top learn more footer?

@SaxonF Do you mean this part? Should we link to the 3 articles @cpapazoglou shared? cc @Rolilink 
<img width="663" alt="155743842-6a52b19a-4e7f-4bb7-ae89-f0243609f389" src="https://user-images.githubusercontent.com/1689238/156591396-0d4a13be-cd46-4759-b680-260075153a5c.png">

**Comment by lsl** [comment]
I was thinking this looks like we need three new components based on the woo landing page.

Footer section - background (prop), header, content section (child components)
Link component - label, header, link
Feature component - header, description

The top image would look like
```
<FooterSection title="" style="background color">
    <LinkCard label="" title="" href="" />
    ...
</FooterSection>

<FooterSection title="" style="background color">
    <FeatureCard title="" description="" />
    ...
</FooterSection>
```

I'm not 100% sure these are the best boundaries so feel free to change how you see fit.

**Comment by SaxonF** [comment]
@allilevine yep that section using the shared articles

**Comment by obenland** [comment]
@Rolilink Have you had a chance to get started on this yet? Is there a PR that you could link to?

**Comment by Rolilink** [comment]
@obenland working on this right now would have something by the end of the day @obenland :) !

**Comment by Rolilink** [comment]
Working on this one here, still missing some mobile styles, translations and folder organization but mostly done with the education footer, I think we are still waiting for content on the bottom part. https://github.com/Automattic/wp-calypso/pull/61726

**Comment by Rolilink** [comment]
We will launch the bottom part first but to complete this one this are the PRs to merge 
* #61726 
* #61746 
@allilevine is this ok? or should we close this issue as soon as we launch #61726

**Comment by lsl** [comment]
@SaxonF should these footers be hidden on the search result pages? (i.e. like the browse all pages that ~have~ [had](https://github.com/Automattic/wp-calypso/issues/61748) infinite scroll?)

**Comment by Rolilink** [comment]
There are some changes @cpapazoglou pointed out I need to address on docs will do this one on a follow-up PR when I am back on Monday if no one address them first
- https://github.com/Automattic/wp-calypso/pull/61726#discussion_r823716271
- https://github.com/Automattic/wp-calypso/pull/61726#discussion_r823718013

**Comment by SaxonF** [comment]
> @SaxonF should these footers be hidden on the search result pages? (i.e. like the browse all pages that ~have~ [had](https://github.com/Automattic/wp-calypso/issues/61748) infinite scroll?)

For now yes
